src/dosbox.cpp

```diff
@@ -581,6 +581,13 @@ void DOSBOX_Init()
 	        "  quiet       |   no    |    no\n"
 	        "  auto        | 'low' if exec or dir is passed, otherwise 'high'");
 
+	Pbool = secprop->Add_bool("always_allow_rw_open", only_at_start, true);
+	Pbool->Set_help(
+	        "Allow host write-protected data files to be read when opened for read-write by\n"
+	        "the DOS program. This avoids generating a false write operation OverlayFS,\n"
+	        "copy-on-write, and network filesystems when the data file(-s) are known to be\n"
+	        "read-only.");
```

Comment:
This is a great and concise description of how it works, @NicknineTheEagle.  For technical users, they can read this and figure out the use-cases.

The benefits can be elusive though (until you actually work with it and see it in action) - I was thinking we should try to describe it from the user-case (or benefits) perspective. 

Some wording here:

*"Many games open all their files with writable permissions; even files that they never modify. This setting lets you write-protect those files while still allowing the game to read them. A second use-case: if you're using a copy-on-write or network-based filesystem, this setting  avoids triggering write-operations for these write-protected files."*

Unfortunately it's a bit wordly, and not sure how to tighten it up. Help wanted.

@NicknineTheEagle , @johnnovak , @FeralChild64 : what do you think?

Reply:
I like @kcgen's version and I don't think it's too wordy. The original description describes the mechanics of the feature accurately and goes into technical details, but doesn't quite make it obvious why would I want to use this feature.